OpenOversight/app/main/views.py

```diff
@@ -896,6 +898,10 @@ def list_officer(
     current_job=None,
     require_photo: Optional[bool] = None,
 ):
+    department = db.session.get(Department, department_id)
```

Comment:
If there isn't a valid `Department` there is no need to perform any of the other queries.